.github/CODEOWNERS

```diff
@@ -16,28 +16,29 @@
 # docs/  docs@example.com
 
 #### Build stuff
-scripts/ @microsoft/fluentui-react-build
-package.json @microsoft/fluentui-react-build
-yarn.lock @microsoft/fluentui-react-build
-*.config.js @microsoft/fluentui-react-build
-*.sh @microsoft/fluentui-react-build
-*.yml @microsoft/fluentui-react-build
-tsconfig.json @microsoft/fluentui-react-build
-lerna.json @microsoft/fluentui-react-build
-.codesandbox @microsoft/fluentui-react-build
-.devops @microsoft/fluentui-react-build
-.eslintrc.* @microsoft/fluentui-react-build
-.gitattributes @microsoft/fluentui-react-build
-.githooks @microsoft/fluentui-react-build
-.gitignore @microsoft/fluentui-react-build
-.npmrc @microsoft/fluentui-react-build
-.npmignore @microsoft/fluentui-react-build
-.vscode @microsoft/fluentui-react-build
-sizeauditor.json @microsoft/fluentui-react-build
+/scripts/ @microsoft/fluentui-react-build
+/package.json @microsoft/fluentui-react-build
+/yarn.lock @microsoft/fluentui-react-build
+/*.config.js @microsoft/fluentui-react-build
+/*.sh @microsoft/fluentui-react-build
+/*.yml @microsoft/fluentui-react-build
```

Comment:
I wasn't sure if these rules were meant to cover **every** *.sh and *.yml file. 

I would recommend going forward that we don't have rules for every *.sh or *.yml file across the entire repo. If there is another subfolder of files that we should add to the core build rules, let me know and I'll add them.

Reply:
The previous config was actually correct; the rules were intended to cover every file of those extensions.

Reply:
What's the reason for applying extra scrutiny beyond the regular folder/package owners for *.sh and *.yml files?

Reply:
The only other `.yml` files should be under `/.devops` (and all of that should be listed as owned by the build team). For scripts and configs, those may require a bit more specialized knowledge to review which is different from what the primary package owner might know about.

Reply:
But if the scripts and configs only affect that package, I think we trust the package owner to make the appropriate decision. I don't disagree that script or configs might want input from folks more familiar with build, but let's leave it up to the package owner to reach out for that more specialized knowledge if they need it. 

I am really trying to reduce the web of cross sign off needed across the repo. If we find that we don't like this setup we can always change it back later.

Does that make sense and address the concern here?